plugins/commands/src/AdminCommands.ts

```diff
@@ -2,6 +2,13 @@ import config from '../config/AdminCommands.config.js'
 import { actions } from '../../actions/Actions.js'
 
 const commands: tm.Command[] = [
+  {
+    aliases: config.updatemaps.aliases,
+    help: config.updatemaps.help,
+    params: [],
+    callback: actions.updateMaps,
```

Comment:
every admin command has an informative message (either public or private depending on the command config) that states in the chat whats been done.

Reply:
Fixed